tests/libtest.sh

```diff
@@ -40,7 +40,7 @@ fi
 # Make sure /sbin/getpcaps etc. are in our PATH even if non-root
 PATH="$PATH:/usr/sbin:/sbin"
 
-tempdir=$(mktemp -d /var/tmp/tap-test.XXXXXX)
+tempdir=$(mktemp -d /run/user/1000/tap-test.XXXXXX)
```

Comment:
Maybe `$UID` instead of `1000`. No all users use uid=1000

Reply:
If this is meant to be in the `$XDG_RUNTIME_DIR`, then use `$XDG_RUNTIME_DIR`, possibly with a fallback (like `${XDG_RUNTIME_DIR:-/var/tmp}/tap-test.XXXXXX` for example).

The most common implementation of the [`XDG_RUNTIME_DIR` spec](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html) is systemd's, which uses `/run/user/$(id -u)`, and there is not really any good reason to use a different path - but the spec intentionally does not guarantee that the path will in fact be `/run/user/$(id -u)`.

Reply:
Apologies, this file had been committed in error.